### ISSUE TITLE
Why store PDFs in Git?

### ISSUE THREAD

**Issue by firegurafiku** [xdanaux/moderncv#31]
Currently the repo size is about 67 MB which is quite heavy for such a small number of revisions. The reason is that PDF files for manual and examples cannot be effectively delta-compressed, so the size increases linearly.

Maybe you should consider removing PDF files from the repo and just write a separate build script which will prepare everything for CTAN submission.

**Comment by FranklinYu** [comment]
[This page from GitHub](https://help.github.com/articles/removing-sensitive-data-from-a-repository/) may help.

**Comment by firegurafiku** [comment]
@franklinyu 

Applying `filter-branch` will invalidate every commit's hash, hence forcing everyone to re-clone their forks. Simple `git rm` would be enough.

**Comment by FranklinYu** [comment]
@firegurafiku I thought you would consider 67 MB already too big. Sorry.

**Comment by firegurafiku** [comment]
@franklinyu 

Yes, I think that 67 MB is already too big. But if the project owner reasonably decides not ruin the hashes, I'll be pretty fine with his decision. The key is to stop further repository growing.
